docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -180,6 +180,8 @@ primary shard that was previously allocated.
 ----
 // NOTCONSOLE
 
+TIP: If a shard is unassigned with an allocation status of `no_valid_shard_copy`, then you should <<fix-cluster-status-recover-nodes,make sure that all nodes are in the cluster>>. If the node containing a primary shard is lost, then you can <<fix-cluster-status-restore,recover the data for the shard>>.
```

Comment:
Extreme pedantry warning: if you lose the node containing a primary shard then normally Elasticsearch will just promote one of the remaining replicas to become the primary without any fuss. We really should talk about losing the in-sync copies here.

```suggestion
TIP: If a shard is unassigned with an allocation status of `no_valid_shard_copy`, then you should <<fix-cluster-status-recover-nodes,make sure that all nodes are in the cluster>>. If all the nodes containing in-sync copies of a shard are lost, then you can <<fix-cluster-status-restore,recover the data for the shard>>.
```